tests/tests.rs

```diff
@@ -85,24 +85,31 @@ fn run_bindgen_tests() {
         }
     });
 
-    // First spawn all child processes and collect them, then wait on each
-    // one. This runs the tests in parallel rather than serially.
-
-    let children: Vec<_> = tests.map(|entry| {
+    let children = tests.map(|entry| {
             let child = spawn_run_bindgen(run_bindgen.clone(), bindgen.clone(), entry.path());
```

Comment:
nit: can you indent this four spaces to the left while you're here? I know it was wrong before.